browser/css/jsdialogs.css

```diff
@@ -2551,3 +2551,13 @@ kbd,
 #ColorPage.jsdialog :is(#iconview_colors, #iconview_recent_colors).ui-iconview .ui-iconview-entry span {
 	font-size: 0; /* Hide placeholder text to prevent layout shifts in color palette iconview widgets due to lazy loading */
 }
+
+/* Writer - Format - Page Style - Columns - Settings */
+
+#TemplateDialog8 #ColumnPage #grid4 {
```

Comment:
number in the id might change with time (next instance)
please match to the beginning of the string:
`[id^="TemplateDialog"] #...`